scripts/postinstall.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict'
+import chalk from 'chalk'
+
+console.warn(
+  '----------------------------------------------------------------------------' +
+    '-----------------------------------------------------------------------\n' +
```

Comment:
This is probably slightly archaic, but for maximum clarity, I'd limit these lines to a maximum of 80 characters so they don't overflow default terminal line lengths.